.claude/agents/security-auditor.md

```
@@ -109,14 +109,15 @@ You receive:
 
 16. **Missing soft-delete filter in a SECURITY DEFINER SELECT** (docs/security.md §15 "Soft-delete in RPCs")
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 11; Actual: 16; Style: 1/2/3

(MD029, ol-prefix)